.storybook/webpack.config.js

```diff
@@ -0,0 +1,136 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+const path = require('path');
+const webpack = require('webpack');
+const { readdir } = require('fs').promises;
```

Comment:
I had node v8 as my default and this threw an error, `lts/*` is supposed to be 10 but nvm didn't pickup the difference when i `cd`'ed into your branch. It also warns in node 10 that this is an experimental feature should we rely on this?

Reply:
You can just use:

```
const { promisify } = require('util')
const fs = require('fs')
const readdirAsync = promisify(fs.readdir)
```